### DISCUSSION TITLE
Change default sort order of containers. Priorized managed containers or hide unmanaged containers.

### DISCUSSION THREAD

**Discussion by mrcozzens** [louislam/dockge#232] [Feature Request]
### ⚠️ Please verify that this feature request has NOT been suggested before.

- [X] I checked and didn't find similar feature request

### 🏷️ Feature Request Type

UI Feature

### 🔖 Feature description

In the container list, I'd like to have containers managed by Dockge at the top of the container list.
Currently, there is very little you can do with containers unmanaged by Dockge.

For me it would be better to have unmanaged containers listed below containers I'm managing with Dockge.
Managed by Dockge Yes/No should be sorted prior to name.  ( With Yes being first of course )
Unmanaged need to be below actively managed containers, or optionally not visible at all.

**My Current Scenario:**
I have 20+ containers, most of which aren't managed by Dockge (since i'm just trying it out).
Currently I have to scroll to find my test container as it is near the bottom alphabetically. Which is slightly annoying when tinkering.

### ✔️ Solution

The preferred sorted container list order:
1.  YES Managed by Dockge containers are at the top of the list, sorted alphabetically.
2.  NOT Managed by Dockge are the next tier. alphabetically.

sql style rough pseudo:   "order by managedYN DESC, name ASC"

Keeping it simple, just add Managed by Dockge YesNo to sort order before alphabetic.
thanks.
AC

### ❓ Alternatives

**ALTERNATIVE THOUGHTS/APPROACH:**
-1  If considering the enhanced sort approach, then this could maybe be a configuration setting defaulted off, as some may find it confusing or undesireable.
-2  A button allowing users to Hide containers not managed by Dockge would also accomplish the same thing.

### 📝 Additional Context

![image](https://github.com/louislam/dockge/assets/6621310/a959215a-cc53-4232-b4bb-54766e13d4a5)

PREFERRED LIST ORDER,  Managed by Dockge First(then alphabetic).  Then unmanaged alphabetically.

* **MANAGED BY DOCKGE, ALPHABETIC**  
1. searxng  ( active, managed )
1. hello-world  ( inactive managed )
* **NOT MANAGED BY DOCKGE**
3. readarr  ( unmanaged, active )
1. remmina    ( unmanaged, active )
1. rimgo    ( unmanaged, active )
1. stirling    ( unmanaged, active )
1. trilium    ( unmanaged, active )

**Comment by louislam** [comment]
https://github.com/louislam/dockge/pull/228

I just updated the sorting yesterday.

**Reply by mrcozzens** [reply]
That's great.  Looks like that merge will do the trick thanks.  
I think the change hasn't hit the docker image quite yet, but i'll eagerly keep an eye out for sort-tweak next time I see the container is updated. 

Appreciate it.

A "Hide unmanaged containers"  button may still be nice-to-have, but #228 sort improvement solves my primary concern.
